### ISSUE TITLE
Import-Excel doesn't import when run through a Task in Task Scheduler

### ISSUE THREAD

**Issue by hansworst24** [dfinke/ImportExcel#423]
Hello,

I'm struggling with a simple script that, when run manually, produces a CSV version of the XLSX that I import, but when run through Windows Task Scheduler, produces an empty file.

I run the script on a Win2008R2 server, with sufficient credentials in the scheduled task. The action in the task is 'powershell.exe -noprofile <file.ps1>'

When the lines below are ran manually, linecount.txt shows the amount of lines imported by Import-Excel. When run from a scheduled task, it shows 0. Therefore the export-csv command creates a empty file.

Can somebody replicate the issue, and find a solution ? I'm truly at a lost and it would be key for me to have this working properly. I'm running the latest Import-Excel module (5.1.1.)

```
$a = Import-Excel c:\temp\Prijslijst.xlsx -WorksheetName 'Afspraken'
$a.count | Out-File c:\temp\linecount.txt
$a | export-csv c:\temp\prijslijst.csv -NoTypeInformation -Delimiter ";"
```

**Comment by Penkethboy** [comment]
make sure <file.ps1> has the full path to the file
make sure "Start In" option of the schedule task is set

why is the -noprofile option being used?

**Comment by hansworst24** [comment]
The filepath is complete, it does also create a "linecount.txt" file, proof that the script is executed properly, therefore it isn't a problem of path, or "Start In"

The -noprofile option was recommended for scheduled tasks, to be a bit more speeder, but with or without it doesn't make any difference.

**Comment by hansworst24** [comment]
Thanks for testing 👍 . I didn't mention 1 key thing : The scheduled task is ran under a different user account ( a service account ), so different from the logged on user. Having said that, I never tried to log on to the server with this serviceaccount, make a profile and see if that resolves the issue. The script should run with or without a user logged on, so not interactively.. I'll make some more tests.

**Comment by hansworst24** [comment]
Allright, I found the cause.I actually launched Powershell in the Task Scheduler from within SYSWOW64, which is 32-bit. When run manually from the console I actually used 64-bit Powershell... And I only installed ImportExcel in 64-bit powershell :)

So the module was never installed in 32-bit environment. Strangely enough the Task Scheduler job still returned return value 0, so I never got triggered that the command wasn't run.

User error in the end ! @Penkethboy  thanks for testing !
